File: 932-monotonic-array/monotonic-array.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var isMonotonic = function(nums) {
    let Bool = true;
    let l = nums.length;
    let first = nums[0]
    let last = nums[l-1]
    if (l<=1)
    return Bool;
            if(first>last){
                for(let i=0;i<l;i++){
                    if(nums[i]<nums[i+1]){
                        Bool = false;
                        return Bool;
                    }
                }
            }
            else{
                for(let i=0;i<l;i++){
                    if(nums[i]>nums[i+1]){
                        Bool = false;
                        return Bool;
                    }
                }
            }
return Bool;
}

File: 932-monotonic-array/monotonic-array.py
class Solution:
    def isMonotonic(self, nums: List[int]) -> bool:
        if len(nums)<=1:
            return True
        else:
            first = nums[0]
            last = nums[len(nums)-1]
            if first > last:
                for i in range(0,len(nums)-1,1):
                    if nums[i] < nums[i+1]:
                        return False
            else:
                for i in range(0,len(nums)-1,1):
                    if nums[i] > nums[i+1]:
                        return False
        return True
